hedyorg/hedy
# 💻 Add hide tables label

Yesterday while on the meeting @gisellandrade  pointed out that the button always said: "View [table]" so in this PR I'm correcting it. As well as simplifying a bit the functions that handle these buttons and replace the use of JQuery.

Fixes #5743


**How to test**

* Log in as a teacher
* Go to the For teachers page and hide and show the tables. The labels should change depending on the state of the table.
* Also try reloading and seeing if the table keeps the same state as previously.
If you're unsure about any of these, don't hesitate to ask. We're here to help!
